packages/react/src/components/DetailsList/DetailsList.test.tsx

```diff
@@ -512,8 +512,10 @@ describe('DetailsList', () => {
     columns[1].onColumnResize = jest.fn();
 
     safeMount(<DetailsList items={mockData(2)} columns={columns} onShouldVirtualize={() => false} />, () => {
-      expect(columns[0].onColumnResize).toHaveBeenCalledTimes(1);
-      expect(columns[1].onColumnResize).toHaveBeenCalledTimes(1);
+      setTimeout(() => {
```

Comment:
Kind of confusing to me why the setTimeout is needed in the test. If you run `jest.runAllTimers` or whatever, and then synchronously run the expects, shouldn't that have the same result?

If there's a good reason for the setTimeout, probably should add a comment to explain when it will be ran. Otherwise, I'd expect the test to simply "pass" after synchronous execution.

Reply:
Good point.  I'll update to the runalltimers and add a comment too.  I was following the approach from another test.